index.html

```diff
@@ -0,0 +1,163 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">
+<html>
+<head>
+	<title>Resume</title>
+	<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+	<link rel="stylesheet" type="text/css" href="css/index.css">
+
+</head>
+<body>
+	<div id = "main">
+		<div id = "header">
+			<div id = "header-content">
+				<div id = "header-content-border">
+					<h1 id = "header-content-border-name">TEAM BLUE</h1>
+					<span id = "header-content-border-designation">Interior Designer</span>
+				</div>
+			</div>
+		</div>
+		<div id = "body">
```

Comment:
id = "body"

dont use extra spaces and provide a proper naming convension
id="main-container"
something like this